bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/ContentAccessHandler.java

```diff
@@ -218,6 +221,8 @@ public Response doHead( final String packageType, final String type, final Strin
 
         if ( path == null || path.equals( "" ) || path.endsWith( "/" ) || path.endsWith( LISTING_HTML_FILE ) )
         {
+            //FIXME: As directory content listing in doGet has switched to use new way, I'm not sure how to change
```

Comment:
I think keeping the HEAD handling consistent with the GET handling is a good idea. I guess that would mean doing a HEAD response in the REST handler for browsing. I'm not sure how much more work that is.

Reply:
Ok. Will do.